File: client/clientUI/ServerApi/Model/TeamDto.java
package cz.cvut.fit.game_viewer.api.model;

import cz.cvut.fit.game_viewer.domain.DomainEntity;

import java.util.List;

public class TeamDto implements DomainEntity<Long> {
  Long id;
  String name;
  String shortname;

  public Long getId() {
    return id;
  }

  public void setId(Long id) {
    this.id = id;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public String getShortname() {
    return shortname;
  }

  public void setShortname(String shortname) {
    this.shortname = shortname;
  }
}

File: client/clientUI/ServerApi/Model/TeamDto.cs
using clientUI.Model;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace clientUI.ServerApi.Model
{
    public class TeamDto : DomainEntity<long?> {
        [JsonProperty]
        private long? id;
        public string name;
        public string shortname;

        public TeamDto(long? id, string name, string shortname)
        {
            this.id = id;
            this.name = name;
            this.shortname = shortname;
        }

        public long? getId()
        {
            return id;
        }

        public void setId(long? id)
        {
            this.id = id;
        }

    }
}
